docs/kb/integrations/wiz.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: wiz
+append_help_link: true
+title: View Semgrep findings in Wiz's Security Graph
+hide_title: true
+description: "Learn how to view Semgrep findings in Wiz's Security Graph."
+tags:
+ - Wiz
+ - Semgrep AppSec Platform
+---
+
+# View Semgrep findings in Wiz's Security Graph
+
+Semgrep integrates with Wiz by calling Wiz’s GraphQL API endpoints and uploading your static analysis (SAST) vulnerability findings to a dedicated Amazon S3 bucket. Your Semgrep SAST vulnerability findings are mapped to the same correlated repository scanned by Wiz and enriched by any available inventory and runtime-related data, such as clusters, pods, containers, cloud configurations, and more. Semgrep's goal is to give you a holistic view of your code and infrastructure security so that you can focus on what matters most.
+
+## Prerequisites and requirements
+
+This integration is available for users with both a [Semgrep Code license](https://semgrep.dev/products/semgrep-code/) and a [Wiz Code Security license](https://www.wiz.io/platform/wiz-code). 
```

Comment:
We need to add the following clause too:

Your SCM needs to be connected to Semgrep. The instructions for this can be found [here](https://semgrep.dev/docs/deployment/connect-scm). The Wiz integration only currently works for GitHub Cloud, GitHub Enterprise Server, Gitlab Cloud, and GitLab Self-Managed. This is because these are the only ones supported by Wiz ([reference](https://win.wiz.io/docs/sast-app-vuln-findings-schema)).

<img width="974" alt="image" src="https://github.com/user-attachments/assets/ee46391b-f2dd-4f0d-8c60-cc319fe027d7" />

Reply:
The second part can also probably live in limitations but we need to make it clear that it's a constraint on their end. We can support all the SCM types if they accept those values.